tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js

```diff
@@ -118,8 +118,6 @@ describe("Feature: Hub and Spoke", () => {
     it("When the user returns to the Hub, Then the section should be marked as 'Completed'", () => {
       $(EmploymentTypeBlockPage.submit()).click();
       expect($(HubPage.summaryRowState("employment-section")).getText()).to.contain("Completed");
-
-      expect($(HubPage.summaryRowTitle("employment-section")).getAttribute("class")).to.contain("summary__item-title--has-icon");
```

Comment:
Should there not be a new assert to check the icon?. I know we unit test the context but should the functional test also check it given the DS can break?

Reply:
I originally set out to replace the css class check with a test, but decided it was for the design system to test as we don't do any visual testing in the functional tests.